cmd/crio/wipe.go

```diff
@@ -195,7 +201,36 @@ func (c ContainerStore) deleteContainer(id string) {
 		logrus.Errorf("Unable to delete container %s: %v", id, err)
 		return
 	}
-	logrus.Infof("Deleted container %s", id)
+
+	if err := c.removeLogSymlinks(id); err != nil {
+		logrus.Errorf("Unable to remove log symlink for container %s: %v", id, err)
+		return
+	}
+}
+
+func (c ContainerStore) removeLogSymlinks(id string) error {
+	files, err := ioutil.ReadDir(kubeletLogDir)
+	if err != nil {
+		return err
+	}
+
+	re := regexp.MustCompile(`\S.+-(\S{64})\.log`)
+
+	for _, f := range files {
```

Comment:
I'm wondering why we have to iterate at all. Can we just construct the logfile name from the ID and then try to remove it?

Reply:
Makes sense.